### PR TITLE
SXT Babel plugin: Transform helpers to direct call

### PR DESCRIPTION
**Pull request by michalsanger**
The plugin replace usage of SX Tailwind helpers with direct SX usage. This way the huge SX Tailwind styles can be omitted from a final bundle.

### CODE REVIEW

File: src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/sxt-aliased/output.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// @flow
+import React, { type Element } from 'react';
```
Comment (tbergq) [inline src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/sxt-aliased/output.js:2]: I thought our lint rule would require a space between `@flow` and the first import statement. 

I am not saying you should fix it, I am just curious as to why it is not reporting error here 🤔

File: src/babel-plugin-transform-sx-tailwind/src/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            if (imported.name === 'tailwind') {
+              tailwindCallee = local.name;
+            } else if (imported.name === 'sxt') {
+              sxtCallee = local.name;
+            }
+          });
+
+          const sxImport = template.ast(`import * as sx from '@adeira/sx'`);
+          path.replaceWith(sxImport);
+        }
```
Comment (tbergq) [inline src/babel-plugin-transform-sx-tailwind/src/index.js:29]: Will it work with `import { create } from '@adeira/sx'` and `import { create as createStyles } from '@adeira/sx'`?

File: src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/sxt-aliased/output.js (hunk 1 above)
Comment (michalsanger) [reply src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/sxt-aliased/output.js:2]: I disabled ESLint for these fixtures in `.eslintignore`

File: src/babel-plugin-transform-sx-tailwind/src/index.js (hunk 2 above)
Comment (michalsanger) [reply src/babel-plugin-transform-sx-tailwind/src/index.js:29]: Good point! Fixed and tested in `fixtures/sx-imported-alias/`

File: src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/sxt-aliased/output.js (hunk 1 above)
Comment (tbergq) [reply src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/sxt-aliased/output.js:2]: That would explain it 💡 any reason for ignoring these files?

Comment (tbergq) [reply src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/sxt-aliased/output.js:2]: Ok, I see, it is necessary, if not the tests will fail from the eslint formatting.

**Review by tbergq — APPROVED** [review]

File: .flowconfig [hunk 3]
```diff
@@ -4,6 +4,7 @@
 <PROJECT_ROOT>/src/docs/build/
 .+/.+\.gzip\.js
 .+/\.next/.+
+<PROJECT_ROOT>/src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures
```
Comment (mrtnzlml) [inline .flowconfig:7]: Would it be possible to make the fixtures valid (real-world examples) to avoid this? 🤔

Comment (michalsanger) [reply .flowconfig]: Actually yes. Not outputs but inputs yes.

**Pull request merged**